modules/world/CMakeLists.txt

```diff
@@ -59,6 +59,10 @@ ocv_module_include_directories()
 #message(STATUS "${OPENCV_MODULE_${the_module}_SOURCES}")
 ocv_create_module(${link_deps})
 
+if(VTK_VERSION VERSION_GREATER_EQUAL "8.90.0")
```

Comment:
What is about using of this check instead: `COMMAND vtk_module_autoinit` ?

We also need checks for:
- enabled viz module itself: `"opencv_viz" IN OPENCV_MODULES_BUILD`
- and viz is part of "opencv_world" `OPENCV_MODULE_opencv_viz_IS_PART_OF_WORLD`

Can we move that into "viz/CMakeLists.txt" ?

Reply:
These are valid points, I'm going to fix it tomorrow. As for moving `vtk_module_autoinit(TARGETS opencv_world MODULES ${VTK_LIBRARIES})` in `viz/CMakeLists.txt` - I don't believe its possible because it requires opencv_world target to exist.

Reply:
I decided to leave `VTK_VERSION` be in order to be consistent with `cmake/OpenCVDetectVTK.cmake`. If `opencv_viz` is a part of world(I added this check), doesnt that mean it is already in `OPENCV_MODULES_BUILD`?

Reply:
We need both checks here.

Reply:
Fixed it.